paraflow-loader-consumer/src/main/java/cn/edu/ruc/iir/paraflow/loader/consumer/threads/DataFlushThread.java

```diff
@@ -64,6 +70,27 @@ public void run()
 
     private void flushMeta(BufferSegment segment)
     {
-        System.out.println(segment.getFilePath());
+        String topic = segment.getFiberPartitions().get(0).getTopic();
+        int indexOfDot = topic.indexOf(".");
+        String dbName = topic.substring(0, indexOfDot);
+        int length = topic.length();
+        String tblName = topic.substring(indexOfDot + 1, length);
+        int fiberValue;
+        String path = segment.getFilePath();
+        long beginTime = segment.getTimestamps()[0];
+        long endTime = segment.getTimestamps()[0];
+        for (long timeStamp : segment.getTimestamps()) {
+            if (timeStamp < beginTime) {
+                beginTime = timeStamp;
+            }
+            if (timeStamp > endTime) {
+                beginTime = timeStamp;
+            }
+        }
+        for (int i = 0; i < segment.getFiberPartitions().size(); i++) {
+            fiberValue = segment.getFiberPartitions().get(i).getFiber();
+            metaClient.createBlockIndex(dbName, tblName, fiberValue, beginTime, beginTime, path);
```

Comment:
The metadata is not correct.
Each fiber records its own begin time and end time inside the segment. Not the segment-level begin time and end time.
Again, the second `beginTime` is not correct neither.